packages/ember-htmlbars/tests/helpers/with_test.js

```diff
@@ -281,122 +32,3 @@ QUnit.module('{{#with}} helper binding to view keyword', {
 QUnit.test('{{with}} helper can bind to keywords with \'as\'', function() {
```

Comment:
Confirmed that `context` in a view means something and there appears to be some rules of context lookup and then fallback to a property on the class or vice versa. 
<img width="1434" alt="screen shot 2016-02-10 at 2 54 32 pm" src="https://cloud.githubusercontent.com/assets/183799/12964778/d69fbd54-d006-11e5-9e48-72a8cd0c702f.png">